desc/vmec.py

```diff
@@ -1782,36 +1812,80 @@ def compute_coord_surfaces(cls, equil, vmec_data, Nr=10, Nt=8, Nz=None, **kwargs
         t_nodes = t_grid.nodes
         t_nodes[:, 0] = t_nodes[:, 0] ** 2
 
+        sym = "lmnc" not in vmec_data.keys()
+
         v_nodes = cls.compute_theta_coords(
             vmec_data["lmns"],
             vmec_data["xm"],
             vmec_data["xn"],
             t_nodes[:, 0],
             t_nodes[:, 1],
             t_nodes[:, 2],
+            lmnc=vmec_data["lmnc"] if not sym else None,
         )
 
         t_nodes[:, 1] = v_nodes
+        if sym:
```

Comment:
I don't think you actually need this if statement? It looks like the only difference between the two cases is the `sym` argument, so you can just call it with `sym=sym`

Reply:
The way `vmec.interpolate` is made, `sym` lets you return the cos and sin sym modes independently, `sym=False` will add the two. I did not want to change the logic of it, so I think this if statement is needed